getting-started/languagedev_participation.html

```diff
@@ -139,7 +139,8 @@ <h2><a href="#tests">More information</a></h2>
 <div class="section">
 <h2 id="contacts"><a href="#contacts">Contacts</a></h2>
 			<ul>
-				<li><a href="mailto:ishida@w3.org">Richard Ishida (ishida @ w3. org)</a>, i18n Lead</li>
+				<li><a href="mailto:xfq@w3.org">Fuqiao Xue (xfq @ w3. org)</a>, i18n Lead</li>
+				<li><a href="mailto:ishida@w3.org">Richard Ishida (ishida @ w3. org)</a>, I18n Core Working Group Staff Contact</li>
 				<li><a href="mailto:addisoni18n@gmail.com">Addison Phillips (addisoni18n @ gmail.com)</a>, I18n Core Working Group Chair</li>
 				<li><a href="mailto:duerst@it.aoyama.ac.jp">Martin Dürst (duerst @ it.aoyama.ac.jp)</a>, i18n Interest Group Chair</li>
```

Comment:
Let's be consistent about capitalization of **I18N**/**i18n**/**I18n**. (Personally, I prefer `I18N`, but I'm probably in the minority)

I would also spell out Internationalization when used with Working Group or Interest Group.

Finally, be consistent when obfuscating email addresses. Here was have `gmail.com` but `w3. <SPACE> org`

Reply:
> Let's be consistent about capitalization of **I18N**/**i18n**/**I18n**. (Personally, I prefer `I18N`, but I'm probably in the minority)

Personally, I prefer 'i18n', mostly because a lack of distinction between the uppercase 'I' and lowercase 'l' in most sans-serif typefaces.

I expanded the abbreviation.

> I would also spell out Internationalization when used with Working Group or Interest Group.
> 
> Finally, be consistent when obfuscating email addresses. Here was have `gmail.com` but `w3. <SPACE> org`

Done.